service/modules/process_result.py

```diff
@@ -0,0 +1,194 @@
+"""Process result file module"""
+import os
+import json
+import datetime
+import logging
+import ast
+import jsend
+import falcon
+import pytz
+import pysftp
+import sendgrid
+import sentry_sdk
+import pandas as pd
+from .permit_applications import PermitApplication
+from ..resources.export import Export
+
+ERROR_GENERIC = "Bad Request"
+ERROR_ACCESS_401 = "Unauthorized"
+
+class ProcessResultFile():
+    """ Class for processing result file """
+    data_file_path = os.path.dirname(__file__) + '/exported_data/'
+
+    def on_get(self, req, resp):
+        """ on get request """
+        try:
+            if req.params['token'] != os.environ.get('EXPORT_TOKEN') and req.params['token'] != os.environ.get('ACCESS_KEY'):
+                raise ValueError(ERROR_ACCESS_401)
+
+            timezone = pytz.timezone('America/Los_Angeles')
+            current_time = datetime.datetime.now(timezone)
+            file_name = 'DBI_permits_' + str(current_time.year) + str(current_time.month) + str(current_time.day)
+            #DBI_permits_YYYYMMDDHHMI_response.csv  where HH = 24 hour clock Mi  = minutes
+            file_name = 'PTS_Export_09_26.csv'
```

Comment:
Is this value hardcoded for testing?

Reply:
yes, we're waiting for final confirmation from MIS